index.md.j2

```diff
@@ -0,0 +1,49 @@
+- [Tools](#tools)
+{%- for repo in tools|sort(attribute='name') %}
+  - [{{ repo.name }}](#{{ repo.name.lower() }})
+{%- endfor %}
+- [Providers](#providers)
+{%- for repo in providers|sort(attribute='name') %}
+  - [{{ repo.name.replace("terraform-provider-", "") }}](#{{ repo.name.lower() }})
+{%- endfor %}
+- [Modules](#modules)
+{%- for provider in modules.keys()|sort %}
+  - [{{ provider }}](#{{ provider.lower() }})
+{%- endfor %}
+
+# Tools
+{%- for repo in tools|sort(attribute='name') %}
+
+## [{{ repo.name }}]({{ repo.url }})
```

Comment:
repo.url doesn't return the public url (but API one)

Reply:
Oops, fixed. Thanks.